src/components/SecondaryPanes/Frames/FrameMenu.js

```diff
@@ -6,37 +6,53 @@ import type { ContextMenuItem } from "debugger-html";
 import { kebabCase } from "lodash";
 
 function formatMenuElement(
-  labelString: string,
-  accesskeyString: string,
+  label: string,
+  accesskey: string,
   click: Function,
   disabled: boolean = false
 ): ContextMenuItem {
-  const label = L10N.getStr(labelString);
   const id = `node-menu-${kebabCase(label)}`;
   return {
     id,
     label,
-    accesskey: L10N.getStr(accesskeyString),
+    accesskey,
     disabled,
     click
   };
 }
 
 function copySourceElement(url) {
-  return formatMenuElement("copySourceUrl", "copySourceUrl.accesskey", () =>
-    copyToTheClipboard(url)
+  return formatMenuElement(
+    L10N.getStr("copySourceUrl"),
```

Comment:
I bet we can simplify this further too: `formatMenuElement("copySourceUrl", () => copyToTheClipboard(url))`